app/views/games/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<br /><br /><br />
+
+<div class = "game-board group">    
+  <% board = [0,1,0,1,0,1,0,1],[1,0,1,0,1,0,1,0] %>

```

Comment:
The controller should contain the variable that just gets passed to the view. I don't think we should define variables within the view logic.